Lib/defconQt/glyphView.py

```diff
@@ -677,7 +959,6 @@ def __init__(self, path, component, parent=None):
         super(ComponentItem, self).__init__(path, parent)
         self._component = component
         self.setTransform(QTransform(*component.transformation))
-        self.setBrush(QBrush(componentFillColor))
```

Comment:
it's done in addComponent now, so that the component can be styled from a common method